GitUp/Application/AppDelegate.m

```diff
@@ -335,6 +335,8 @@ - (void)applicationWillFinishLaunching:(NSNotification*)notification {
   [Crashlytics startWithAPIKey:@"946dcb56f0db1be8f0c52df6e7d392202a2cc9b2"];
   [[Crashlytics sharedInstance] setDelegate:self];
 #endif
+  
+  [[NSAppleEventManager sharedAppleEventManager] setEventHandler:self andSelector:@selector(_getUrl:withReplyEvent:) forEventClass:kInternetEventClass andEventID:kAEGetURL];
```

Comment:
Ya, it wasn't registering early enough if I placed the code there in cases where the application wasn't already open.